src/polymer/expression-scanner.ts

```diff
@@ -120,6 +126,17 @@ export abstract class DatabindingExpression {
       return;
     }
     let expression = expressionStatement.expression;
+    if (limitation === 'identifierOnly') {
+      if (expression.type !== 'Identifier') {
+        this.warnings.push(this._validationWarning(
+            `Expected just a name here, not an expression`, expression));
+      }
+    } else if (limitation === 'callExpression') {
+      if (expression.type !== 'CallExpression') {
+        this.warnings.push(this._validationWarning(
```

Comment:
Minor nit; maybe this could be less nesty?  Unless theres an intention to extend the cases within the limitation categories later...?
```
if (limitation === 'identifierOnly' && expression.type !== 'Identifier) {
  ...
} else if (limitation === 'callExpression' && expression.type !== 'CallExpression') {
  ...
}
```